Traceback (most recent call last):
  File "/Users/dl20/opt/anaconda3/envs/jupyterbook/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/dl20/opt/anaconda3/envs/jupyterbook/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/dl20/opt/anaconda3/envs/jupyterbook/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/dl20/opt/anaconda3/envs/jupyterbook/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/dl20/opt/anaconda3/envs/jupyterbook/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/Users/dl20/opt/anaconda3/envs/jupyterbook/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/dl20/opt/anaconda3/envs/jupyterbook/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/dl20/opt/anaconda3/envs/jupyterbook/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Turtles don't come in basic Python, we need to get them from a "module"
from mobilechelonian import Turtle

terry = Turtle()     # create a turtle, and call him terry

# terry will trace out the path he moves along, like he has a pen attached to him
terry.forward(100)   # move terry forward 100 units
terry.left(90)       # turn terry left 90 degrees
terry.forward(100)   # move terry forward 100 units again
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Input [0;32mIn [13][0m, in [0;36m<cell line: 2>[0;34m()[0m
[1;32m      1[0m [38;5;66;03m# Turtles don't come in basic Python, we need to get them from a "module"[39;00m
[0;32m----> 2[0m [38;5;28;01mfrom[39;00m [38;5;21;01mmobilechelonian[39;00m [38;5;28;01mimport[39;00m Turtle
[1;32m      4[0m terry [38;5;241m=[39m Turtle()     [38;5;66;03m# create a turtle, and call him terry[39;00m
[1;32m      6[0m [38;5;66;03m# terry will trace out the path he moves along, like he has a pen attached to him[39;00m

[0;31mModuleNotFoundError[0m: No module named 'mobilechelonian'
ModuleNotFoundError: No module named 'mobilechelonian'

